MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -482,6 +599,10 @@ enum MSBuildSpecialCommitKind
 		NewGuid,
 		PropertyReference,
 		ItemReference,
+		Element,
```

Comment:
These should be handled on the base XML layer - `XmlCompletionItemKind.Element` etc already exists.